### ISSUE TITLE
Hard coded URL part

### ISSUE THREAD

**Issue by ad-m** [Bouke/django-user-sessions#43]
Hello,

I am would like ask why URL contains hard-coded "account/sessions/"? Is there any reason to leave it other than back-ward compatibility?

**Comment by Bouke** [comment]
Good point, it should be removed, allowing users of the package to define their own URL structure.
